licensing/src/main/java/io/crate/license/LicenseService.java

```diff
@@ -130,7 +137,15 @@ public boolean hasValidLicense() {
         if (enterpriseEnabled == false) {
             return true;
         }
-        return !isLicenseExpired(currentLicense());
+        // We consider an instance that is bound to loopback as a development instance and by-pass the license expiration.
+        // This makes it easier for us to run our tests.
+        return boundToLocalhost() || !isLicenseExpired(currentLicense());
+    }
+
+    private boolean boundToLocalhost() {
+        Predicate<TransportAddress> isLoopbackAddress = t -> t.address().getAddress().isLoopbackAddress();
```

Comment:
minor: does it make sense to keep this reference once at class level instead of doing this on each invocation?
It's not expensive though, therefore I think it's minor.